README.md

```diff
@@ -1 +1,189 @@
+
+
+## 🛠 Built With <a name="built-with"></a>
+* Ruby
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Ruby</a></li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Insert code you want to debug]**
+- **[It returns the decoded message]**
+- **[Built using Ruby]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+ * Ruby installed on your local machine.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  cd my-folder
+  git clone https://github.com/eng-mohamed-eldeeb/ruby-Morse-code.git
+
+### Install
+
+
+
+### Usage
+
+To run the project, execute the following command:
+* cd ruby-Morse-code
+* run the command ruby main.rb
+* see the project running
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 Mohamed Eldeeb 
+
+- [GitHub](https://github.com/eng-mohamed-eldeeb)
+- [Twitter](https://www.linkedin.com/in/mohamed-eldeeb-a69022206/)
+- [LinkedIn](https://twitter.com/eldeeb_3o)
+
+
+👤 **Akpuru Solomon Barine
+
+- GitHub: [@githubhandle](https://github.com/solobarine)
+- Twitter: [@twitterhandle](https://twitter.com/solomon-akpuru)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/solomon-akpuru)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank ...
+
+* Everyone who enabled us complete the project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Will you have any updates in the future?]**
+
+  - [No, we will not have an update for this project.]
+
+- **[Can I clone and use the project?]**
+
+  - [Yes. Feel free to use it]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:**  I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, as your license file in your root directory is named `MIT.md` and not `LICENSE`